src/silicone/time_projectors/extend_latest_time_quantile.py

```diff
@@ -45,6 +46,19 @@ def derive_relationship(self, variable):
             The variable for which we want to calculate timeseries (e.g.
             ``"Emissions|CO2"``).
 
+        smoothing : None, float or string
```

Comment:
This should ideally match the exact text that is used elsewhere (I don't think it does right now...)